packages/core/package.json

```diff
@@ -88,9 +88,10 @@
     "@libp2p/peer-id": "^1.1.10",
     "@multiformats/multiaddr": "^11.0.6",
     "@noble/hashes": "^1.1.3",
-    "@waku/utils": "*",
     "@waku/interfaces": "*",
+    "@waku/peer-exchange": "*",
```

Comment:
core must not depend on the `@waku/peer-exchange` package as it goes against the point of having a core package free from heavy dependencies needed by some Waku Protocols.